mnd/mnd/integration/mims/mims_view.py

```diff
@@ -0,0 +1,107 @@
+import logging
+
+
+from django.http import JsonResponse
+from django.views.decorators.http import require_GET
+
+from .mims_service import MIMSService
+from .mims_cache import (
+    CMIInfo, ProductInfo, get_cache, get_cmi_cache,
+    search_cache, update_cache, update_cache_entry,
+    update_cmi_cache
+)
+
+logger = logging.getLogger(__name__)
+
+
+@require_GET
+def product_search(request):
+
+    def active_ingredient(result):
+        return ", ".join(result.split("+"))
```

Comment:
Could be split by " + " to avoid gaps before commas

Reply:
Updated.